load/LanguageReaderStageTest.py

```diff
@@ -0,0 +1,117 @@
+import os
```

Comment:
This class should be removed because it has been replaced by `load/TestLanguageReaderStage.py.` I have created the next PR to remove it: https://github.com/faithcomesbyhearing/dbp-etl/pull/44